content/news/playtest-20160403.md

```diff
@@ -0,0 +1,118 @@
+
+As usual, both the Tiberian Dawn and Red Alert mod have received some balance changes. Additionally, a number of Red Alert maps have been revised to be more balanced. A complete list of all these changes can be found [in the changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/<hash>).
+
+<div style="text-align:center" markdown="1">
+![Restart dialog](/images/news/20160403-cnc-restart-dialog.png)
+<p>
+Players finally have a restart option after failing a mission or skirmish!
+</p>
+
+![Team colors and health bar improvements](/images/news/20160403-ra-team-colors.png)
+<p>
+When team colors are enabled in the settings, they will now be used on the radar map.<br/>
+You can also set health bars to only appear when units are damaged.
+</p>
+
+![Nod units with proper color](/images/news/20160403-cnc-nod-colors.png)
+<p>
+This is for the nostalgic ones among us (and who isn't?).<br/>
+Nod units in missions now show their proper silver colors known from the original game.
+</p>
+
+![Worm trails](/images/news/20160403-d2k-worm-trail.png)
+<p>
+Worms have received some polish and now leave a trail similar to the one from the original game.
+</p>
+</div>
+
+<hr />
+## Mapping and modding changes
+<hr />
+Maps have seen many changes since the release. So many changes, in fact, that the map format number was incremented four times &ndash; from 7 to 11 &ndash; during this development cycle!
+
+We will cover these changes in more detail in the news post for the third playtest. In the meantime, the only thing you really need to know is that the game won't upgrade maps automatically anymore. Please download new revisions from [the OpenRA Resource Center](http://resource.openra.net) instead.
+
+<div style="text-align:center" markdown="1">
+![Upgrading maps in the resource center](/images/news/20160403-web-resource-center-map-upgrade.png)
+<p>
+Use the Resource Center to upgrade maps easily to new versions of OpenRA.
+</p>
+</div>
+
+Support for third-party mods will significantly improve with the next release. It is now possible to create an archive of your whole mod (called ".oramod"). Installing it on other computers is then just a matter of placing that archive in the "mods" subdirectory of the Support folder.
+
+The news post for the second playtest will detail what changes your mod needs to make it compatible with the upcoming release and to enable the .oramod packaging. All shipped mods have already been converted, so feel free to take a look at those.
+
+In the future, OpenRA will feature an ingame mod browser that will allow you to search for and install mods directly from the Resource Center.
+
+<hr />
+## Dedicated server changes
+<hr />
+Dedicated servers have seen quite a few improvements during this cycle, which make them altogether much easier to run.
+
+<div style="font-family: monospace; border-radius: 4px; background-color: #272d2c; padding:10px; margin:10px;">
+<em>
+$ ./OpenRA.Server.exe<br />
```

Comment:
Could you add some demonstration parameters, like `Game.Mod=ra Server.Name="My Server" Server.AdvertiseOnline=True`? If you reduce the font size for the div to 85% it will fit on one line, and help distinguish the whole block from the rest of the text.  Or substitute other parameters as you want.